src/main/java/com/neonlab/product/repository/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.neonlab.product.repository;
+import com.neonlab.product.entities.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product,String> {
+    Optional<Product> findByCode(String code);
+
+    Boolean existsByCode(String code);
```

Comment:
Why do you need this?


If the first returns value than exist else not.

It does not seems useful

cc- @sjha24